generator/src/main/java/com/scottlogic/deg/generator/generation/databags/RowSpecDataBagSourceFactory.java

```diff
@@ -1,7 +1,9 @@
 package com.scottlogic.deg.generator.generation.databags;
 import com.scottlogic.deg.generator.fieldspecs.RowSpec;
 
-public interface RowSpecDataBagSourceFactory{
-    DataBagSource createDataBagSource(RowSpec rowSpec);
+import java.util.stream.Stream;
+
+public interface RowSpecDataBagSourceFactory {
```

Comment:
This wouldn't be the right name anymore.

Reply:
Agree, i renamed and undid the renaming, because github couldn't handle the diff

Reply:
I'd go with the rename and highlight it with a GitHub comment. It's better to have clean code, sacrificing some of the clarity in the review than sacrificing good code because of the review tool.